backend/modules/transform/domain/commands.py

```diff
@@ -1,9 +1,10 @@
 from dataclasses import dataclass
 
+import pandas as pd
+
 from ...common.domain import commands as common_commands
-from ...data.domain import value_objects as data_value_objects
 
 
 @dataclass(frozen=True)
 class TransformData(common_commands.DomainCommand):
-    data: data_value_objects.InputData
+    data: pd.DataFrame
```

Comment:
I had to change domain command for Transformation, because Extract produces pandas DataFrame.

Reply:
Please, add this:

pd.DataFrame[InputData]

Reply:
No, it's not true. You will get DataFrame here, you can;t type anything else.

It should be possible via `pandora.typing.panas.DataFrame`

Reply:
I changed it back to data: data_value_objects.InputData

and then casted it in service command 
df: pd.DataFrame = cast(pd.DataFrame, command.data)

please let me know, if this approach is fine for you

Reply:
you are right! it's possible with pandora DataFrame. Thanks